packages/cli/package.json

```diff
@@ -247,7 +247,8 @@
       },
       "s3": {
         "xz": true,
-        "host": "https://cli-assets.heroku.com"
+        "host": "https://cli-assets.heroku.com",
+        "bucket": "heroku-cli-assets"
```

Comment:
We should be able to change this to test deploy process in a different bucket.

Reply:
That will be handy